src/components/common/modal/ModalOverlay.js

```diff
@@ -25,10 +25,10 @@ const getStylesFromProps = ({ theme }) => ({
     width: '100%',
     flexGrow: 1,
     flexShrink: 0,
-    paddingBottom: `${15 * vh}px`,
+    paddingBottom: theme.modals.overlayVerticalPadding + browserPadding * 1.5,
     paddingLeft: theme.modals.overlayHorizontalPadding,
     paddingRight: theme.modals.overlayHorizontalPadding,
-    paddingTop: `${15 * vh}px`,
+    paddingTop: theme.modals.overlayVerticalPadding - browserPadding * 0.5,
```

Comment:
The main problem is the behaviour between devices...on iOS the browser does not change the screen height or the vh when top and bottom bar are visible...but Android consider those bars and change the vh and screen height updating the content to fit inside...
For that reason I need to validate if it is in mobile safari, because that one is the one I need to fix...
I tried to use a custom vh, but if I use the default padding, the modal is resized but it is not centered...so I have to reduce size to make it fit correctly (and it is not going to be centered anyway).
With this solution I'm considering **browserPadding**, which value is 44 (the height of the bottom bar), and it works better. It looks centered in most of the devices...the problem is with iPhone X, because bottom bar is taller than the default one for Safari...so it looks good but it is not 100% centered